spinalcordtoolbox/aggregate_slicewise.py

```diff
@@ -260,12 +260,14 @@ def aggregate_per_slice_or_level(metric, mask=None, slices=[], levels=[], distan
 
     # If user neither specified slices nor levels, set perslice=True, otherwise, the output will likely contain nan
     # because in many cases the segmentation does not span the whole I-S dimension.
+    if perslice:
+        # If perslice is specified, put distance_pmj to None to prioritize perslice
+        distance_pmj = None
```

Comment:
> (As a side-note, the `if perslice is None` code is dead code: there's only one caller of this code, and it always passes in either `0` or `1` (and not `None` or even a boolean), so the `if` condition is always false.)

I checked it out a bit, some tests in the test suite fails if I change it to : `if not perslice`, I think we should change it maybe in another PR since it will require some refractoring elsewhere too (and not related to this PR's changes)